modules/ve-cx/init/ve.init.mw.CXTarget.js

```
@@ -526,6 +526,16 @@ ve.init.mw.CXTarget.prototype.onSurfaceReady = function () {
 
 	if ( this.translation.hasTranslatedSections() ) {
 		this.targetSurface.$element.addClass( 've-ui-cxTargetSurface--non-empty' );
+	} else {
+		// Don't start auto translations if there are Translated Sections;
+		// By: Ibrahem Qasim
+		const sections = $('.cx-column--translation article').find('section');
```

ESLint

[error] jQuery collection names must match the variablePattern

(no-jquery/variable-pattern)